### PR TITLE
[Refactor] Resolving merge conflicts between develop and chat feature

### PR DESCRIPTION
**Pull request by JoshuaLai**
## Purpose
Back merge develop into chat feature branch 

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes
- [ ] Other... Please describe:

### CODE REVIEW

**Review by vhuseinova-msft — APPROVED** [review]

**Review by jpeng-ms — APPROVED** [review]

**Review by jimchou-dev — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift [hunk 1]
```diff
@@ -16,12 +16,16 @@ class ControlBarViewModel: ObservableObject {
     @Published var cameraPermission: AppPermission.Status = .unknown
     @Published var isAudioDeviceSelectionDisplayed: Bool = false
     @Published var isConfirmLeaveListDisplayed: Bool = false
+    @Published var isMoreCallOptionsListDisplayed: Bool = false
+    @Published var isShareActivityDisplayed: Bool = false
 
     let audioDevicesListViewModel: AudioDevicesListViewModel
-
     var micButtonViewModel: IconButtonViewModel!
```
Comment (emlynmac) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift:23]: I thought we had swift lint banning force-unwrapped optionals?

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/ViewComponents/Drawer/MoreCallOptionsListViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import AVFoundation
```
Comment (emlynmac) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/ViewComponents/Drawer/MoreCallOptionsListViewModel.swift:7]: Why does this include AVFoundation?

Comment (JoshuaLai) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/ViewComponents/Drawer/MoreCallOptionsListViewModel.swift:7]: I think you are right. Good call out we will clean this up in a refactor task. cc @vhuseinova-msft

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift (hunk 1 above)
Comment (JoshuaLai) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/SwiftUI/Calling/CallingViewComponent/ControlBarViewModel.swift:23]: Something we will need to look into the swift linter rules. I will follow up on this.

**Pull request merged**